app/layout/account-bar.jsx

```diff
@@ -62,7 +62,7 @@ export default class AccountBar extends React.Component {
         messageCount: unread_count,
         unread: unread_messages
       });
-    });
+    }).catch((err) => { console.error(err) });
```

Comment:
Whoops this was left in by accident because the "you're not allowed in this conversation" errors kept bugging me. This should be a separate PR, to be honest.